scripts/exhafs_obs_proc.sh

```diff
@@ -100,6 +100,8 @@ dropsondetarfile=${COMINhafs_OBS}/hafs.t${cyc}z.dropsonde.tar
 # Proceed if dropsondetarfile exists and non-empty
 if [ -s ${dropsondetarfile} ]; then
   ${TAR} -xvf ${dropsondetarfile}
+  #Remove bad data (not containing REL or SPG)
+  grep -lLE --null 'REL|SPG' ./*.mod | xargs -0 rm
```

Comment:
I assume this change will impact the observation preprocessing even if online satellite bias correction isn't used. Would you provide some more information about why you are making this change? Why are non-REL, non-SPG data bad?

Reply:
This change addresses some bad dropsonde data files that causes the obs_proc step to fail. The data files should have been filtered out by QC but weren't for some reason. The fix here simply deletes the data, but I have a slightly different code in my working HAFS directory that moves the files here, so you can see for yourself:
/work/noaa/aoml-hafsda/knisely/bad_drop_data

Reply:
Thanks for the example files @jak5808. It is obvious that they contain bad data (or, really, no data at all). Probably @BinLiu-NOAA can comment on whether this is the best approach for dealing with this issue.

Reply:
If you like, I can remove this change until after our branch is rolled into develop